clang/lib/CIR/CodeGen/CIRGenCall.cpp

```diff
@@ -18,22 +18,118 @@
 using namespace clang;
 using namespace clang::CIRGen;
 
-CIRGenFunctionInfo *CIRGenFunctionInfo::create(CanQualType resultType) {
-  void *buffer = operator new(totalSizeToAlloc<ArgInfo>(1));
+CIRGenFunctionInfo *
+CIRGenFunctionInfo::create(CanQualType resultType,
```

Comment:
This same change is being made in https://github.com/llvm/llvm-project/pull/136854.

@el-ev I think it makes sense to hold your change until this one has been merged. That will help with the concern I had about limited testing in your PR.